### ISSUE TITLE
API Landing Page not advertise conformance classes correctly

### ISSUE THREAD

**Issue by philvarner** [stac-utils/stac-fastapi#169]
The landing page (root endpoint) does not have a conformsTo field, and the conformance classes in /conformance do not correspond to STAC API 1.0.0-beta.1 or 2. https://github.com/radiantearth/stac-api-spec/blob/master/overview.md#conformance-classes

This is an invalid STAC API as it works currently.

Planetary Computer does have a conformsTo field.

**Comment by moradology** [comment]
This should be addressed as a result of changes in #181 - the updated `stac-pydantic` Landing model includes conformance classes: https://github.com/stac-utils/stac-fastapi/pull/181/files#diff-fe72d159853c2c83c51a4c09af0905cbc8706b18f8fd42ca2491a8c02dc37c18R49

**Comment by philvarner** [comment]
I see that conformsTo exists now, but the classes are invalid.

```
"conformsTo": [
    "https://stacspec.org/STAC-api.html",
    "http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#ats_geojson"
  ],
```

should be like:

```
"conformsTo": [
    "https://api.stacspec.org/v1.0.0-beta.2/core",
    "https://api.stacspec.org/v1.0.0-beta.2/ogcapi-features",
    "https://api.stacspec.org/v1.0.0-beta.2/item-search"
  ],
```

**Comment by moradology** [comment]
Makes sense. It seems like the analogous conformance classes aren't available at e.g. https://api.stacspec.org/v1.0.0/core - do you by chance know of their whereabouts? `master` should be past beta.2 now

**Comment by philvarner** [comment]
https://api.stacspec.org/v1.0.0-beta.2/core/

1.0.0-beta.2 is the current version of STAC API.  1.0.0-beta.3 will be released soon.

**Comment by moradology** [comment]
Yeah, looks like I confused beta with rc 🤦

**Comment by moradology** [comment]
@philvarner I *believe* things are now addressed in #214

**Comment by philvarner** [comment]
It looks right to me!
